clang/runtime/dpct-rt/include/dpct/group_utils.hpp

```diff
@@ -0,0 +1,241 @@
+//==---- group_utils.h ------------------*- C++ -*---------------==//
+//
+// Copyright (C) Intel Corporation
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+// See https://llvm.org/LICENSE.txt for license information.
+//
+//===----------------------------------------------------------------===//
+
+#ifndef __DPCT_DPCPP_EXTENSIONS_H__

```

Comment:
The include guard name should be changed since it is currently mirroring the macro name in the `dpcpp_extensions.h` file.